### PR TITLE
(Candid-based) Web UI

### PR DESCRIPTION
**Pull request by matthewhammer**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// todo
```
Comment (chenyan-dfinity) [inline index.js]: This file is suppose to be under `web/`. If you want to put it here, you need to update `entrypoint` in `dfx.json`.

**Comment by chenyan-dfinity** [comment]
With dfx 0.5.3, you can get the web UI without any frontend stuff. My commit removes all the patches I gave earlier, and CI is passing :)

**Comment by matthewhammer** [comment]
@chenyan-dfinity Oops, sorry that I missed this commit and message.  I'll check this out.

**Comment by matthewhammer** [comment]
Just verified this works locally.  Awesome!

**Pull request merged**